README.md

```diff
@@ -79,6 +79,14 @@ var address = Address(await adapter.find('addresses', '1'));
 
 Will send the request: `GET /addresses/1`.
 
+#### Retrieving a relationship
+
+```dart
+var address = Address(await adapter.find('addresses', '1', queryParams: {'include', 'state'}));
+```
+
+Will send the request: `GET /addresses/1?include=state`.
```

Comment:
@algodave not sure how our internal api will work with that, but I guess, it should be there

Reply:
@Bringoff The `include` query param is part of the `JSON:API` specifications, so it would be great to have our library support it 🙂